### ISSUE TITLE
Plugin is not accessible

### ISSUE THREAD

**Issue by ico85** [PhilippeMarcMeyer/vanillaSelectBox#103]
The Plugin is useless for screenreader users because you cannot reach the dropdown with keyboard navigation only. Also it has no semantic because it has no roles, for example role combobox, role listbox for the dropdown, role option for the options and a lot more missing aria-* Attributes. Also the options are not focusable and you cannot close it via escape. Today it's a good practice to build components accessible. Here is a lot of stuff to be enhanced regarding accessibility.
